docs/lang/articles/contribution/writing_cpp_tests.md

```diff
@@ -21,6 +21,11 @@ make
 ./taichi_cpp_tests
 ```
 
+:::note
+On Windows, `taichi_cpp_tests.exe` will be outputted into the `%TAICHI_REPO_DIR%\bin` directory,
+so you can directly run it after adding the directory to `PATH` in [Setting up Taichi for development](dev_install.md#setting-up-taichi-for-development-3).
```

Comment:
Hopefully I understand how the link works correctly -- feel free to correct me if the syntax is wrong.